watertap/unit_models/reverse_osmosis_0D.py

```diff
@@ -259,198 +206,67 @@ def mass_transfer_phase_comp_initialize(b, t, p, j):
             units=units_meta('mass') * units_meta('time')**-1,
             doc='Mass transfer to permeate')
 
+        # constraints for additional variables (i.e. variables not used in other constraints)
+
+        # Not in 1DRO
+        @self.Constraint(self.flowsheet().config.time,
+                         solute_set)
+        def eq_rejection_phase_comp(b, t, j):
+            return (b.rejection_phase_comp[t, 'Liq', j] ==
+                    1 - (b.mixed_permeate[t].conc_mass_phase_comp['Liq', j] /
+                         b.feed_side.properties_in[t].conc_mass_phase_comp['Liq', j]))
```

Comment:
Ah ok this answers my previous question. The rejection variable not being in the 1DRO is an oversight, it should also have it.